Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -769,15 +774,30 @@ ShaderCode GeneratePixelShaderCode(DSTALPHA_MODE dstAlphaMode, APIType ApiType,
       out.Write("\tdepth = float(zCoord) / 16777216.0;\n");
   }
 
+  // No dithering for RGB8 mode
```

Comment:
This PR doesn't support RGBA8 anywhere, neither does the console, it only supports RGB8.